### PR TITLE
Create MIT LICENSE

### PR DESCRIPTION
**Pull request by Richardng12**
Adding MIT license

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Richard Ng
```
Comment (jhsi517) [inline LICENSE]: Should the name be the Org name? "SE701ProjectGroup4"

Comment (Richardng12) [reply LICENSE]: Yeah i was wondering about that since we don't have a project name

Comment (DragonMeme) [reply LICENSE]: I think Team Hong decided to have the organization name "Divergence Studios".

Comment (Richardng12) [reply LICENSE]: Edited name to SE701ProjectGroup4

**Pull request merged**